l10n_be_codabox/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Niboo SRL (<https://www.niboo.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "CodaBox",
+    "category": "Accounting",
+    "summary": "Connection to CodaBox",
+    "website": "https://github.com/OCA/l10n-belgium",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Niboo SRL,Odoo Community Association (OCA)",
```

Comment:
If you plan to maintain this module actively (hopefully :), do not hesitate to add a `maintainers` key. See [here](https://odoo-community.org/page/maintainer-role) for details.